tests/password_manager/test_edit_login_change_everything.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from selenium.webdriver import Firefox
+
+from modules.page_object_about_pages import AboutLogins
+
+@pytest.fixture()
+def test_case():
+    return "2241121"
+
+def test_edit_mode_change_everything(driver: Firefox):
+    """
+    C2241121 - Edit Mode - Change Everything
+    """
+    # Initialize page object and open AboutLogins page
+    about_logins = AboutLogins(driver).open()
+
+    # Click add login button
+    about_logins.click_add_login_button()
+
+    # Create initial test login data and add new login
+    initial_login = {
+        "origin": "test.mozilla.org",
+        "username": "test_user",
+        "password": "test_password123"
+    }
+    about_logins.create_new_login(initial_login)
+
+    # Click Edit button through shadow DOM
+    assert driver.execute_script("""
+        const editButton = document.querySelector('login-item').shadowRoot.querySelector('edit-button').shadowRoot.querySelector('button.ghost-button');
+        if (editButton) { editButton.click(); return true; }
+        return false;
+    """), "Edit button not found."
```

Comment:
This style of finding elements is nothing like we do for this project, please reference other tests and see how it is done and update yours accordingly.